### DISCUSSION TITLE
How to change just the background color of a cell, leaving all other styling intact???

### DISCUSSION THREAD

**Discussion by sanoanalytics** [qax-os/excelize#1408] [Q&A]
How can I change a cell's background color and leave all other styling intact?

**Comment by xuri** [comment]
Thanks for your feedback. This problem is similar to [#990](https://github.com/xuri/excelize/issues/990#issuecomment-899313624), the library doesn't support append or merge styles with existing styles currently.

**Reply by sanoanalytics** [reply]
Thanks xuri, it would be an enhancement to be able to change a specific style property, such as font, color, border etc. Without other properties being set to default.
